backend/deepcell_label/label.py

```diff
@@ -199,6 +200,24 @@ def clean_cell(self, cell):
         """Ensures that a cell is a positive integer"""
         return int(max(0, cell))
 
+    def clean_labels(self, labeled, cells):
+        # TODO: does this work with time series right now?
+        """Ensures that labels do not include any values that do not correspond
+           to cells (eg. for deleted cells.)
+
+        Args:
+            labeled: numpy array of shape (height, width)
+            cells: list of cells labels like { "cell": 1, "value": 1, "t": 0}
+
+        Returns:
+            (numpy array of shape (height, width), cells with updated values)
+        """
+        values = [cell['value'] for cell in cells]  # get list of values
+        cell_mask = np.in1d(labeled, values)
+        deleted_mask = np.logical_not(cell_mask).reshape(labeled.shape)
+        labeled[deleted_mask] = 0  # delete any labels not in values
```

Comment:
`np.isin` would preserve the shape of labeled so you don't have to reshape after. Both `isin` and `in1d` have an invert parameter that can bypass the `logical_not`.

```
deleted_mask = np.isin(labeled, values, invert=True)
labeled[deleted_mask] = 0
```